src/Microsoft.NetCore.Analyzers/Core/Runtime/DoNotUseEnumerableMethodsOnIndexableCollectionsInsteadUseTheCollectionDirectly.Fixer.cs

```diff
@@ -21,9 +29,51 @@ public sealed override FixAllProvider GetFixAllProvider()
 
         public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
+            var diagnostic = context.Diagnostics[0];
+            var codeAction = CodeAction.Create("Use indexer", c => UseCollectionDirectly(context.Document, context.Span, c));
```

Comment:
> CodeAction.Create

Please use the pattern used in this repo for creating code actions, see https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/AbstractTypesShouldNotHaveConstructors.Fixer.cs#L59

Reply:
Done in the new commit.